scripts/apps/monitoring/views/monitoring-view.html

```diff
@@ -184,7 +184,7 @@ <h2 class="subnav__page-title sd-flex-no-grow" ng-hide="type === 'monitoring'">
                         <div sd-item-sortbar class="sortbar-container"></div>
 
                         <div class="dropdown dropdown--align-right" dropdown ng-if="type === 'monitoring'">
-                            <button area-label="{{!monitoring.viewColumn && (!view || view === 'compact') ? 'List View' : view === 'photogrid' ? 'Photo Grid' : monitoring.viewColumn ? 'Kanban View : ''}}" 
+                            <button area-label="{{!monitoring.viewColumn && (!view || view === 'compact') ? gettext('List View') : view === 'photogrid' ? gettext('Photo Grid') : monitoring.viewColumn ? gettext('Kanban View') : ''}}" 
```

Comment:
Translation string extraction using `gettext` doesn't work from `.html` files - either translate attribute or filter should be used.